### ISSUE TITLE
Preview pane

### ISSUE THREAD

**Issue by WillAbides** [octobox/octobox#209]
What would you think of taking a page from https://notifications.githubapp.com and adding a preview pane?

**Comment by andrew** [comment]
Dependent on #4, which is pretty involved and requires repo scope for private repos

**Comment by jerone** [comment]
Duplicate of https://github.com/octobox/octobox/issues/69.
